src/app/login/page.tsx

```diff
@@ -0,0 +1,22 @@
+import { Wrapper } from "./../../components/wrapper";
+import React from "react";
+import { LoginForm } from "~/app/login/loginForm";
+import { NextPage } from "next/types";
+import { getProviders } from "next-auth/react";
+import { type AppProps } from "next/app";
+import { BuiltInProviderType } from "next-auth/providers";
+import { getServerAuthSession } from "~/server/auth";
+
+const LoginPage = async (): Promise<React.JSX.Element> => {
+  const providers = await getProviders();
+  const session = await getServerAuthSession();
+  return (
+    <main className="min-h-screen bg-[#050816] px-4 py-8">
```

Comment:
Add constant for color in tailwind configuration and use it. 

For naming colors you can use this website - https://chir.ag/projects/name-that-color/

Like here: 
<img width="570" alt="image" src="https://github.com/Exclusive-PG/next-jazz-music-client/assets/37477697/2149e916-1eef-4fdc-ac74-2dcfec8e6f9a">